use-shopping-cart/core/stripe-middleware.js

```diff
@@ -4,21 +4,9 @@ import '@stripe/stripe-js'
 export const handleStripe = (store) => (next) => async (action) => {
   const stripePublicKey = store.getState().stripe
   const cart = store.getState()
-  let stripe
 
   if (action.type === 'cart/redirectToCheckout') {
-    try {
-      stripe = await Stripe(stripePublicKey)
-      // new method provided to help Stripe keep track of usage.
-      stripe.registerAppInfo({
```

Comment:
@gudatcomputers I can't seem to find `stripe.registerAppInfo` anywhere on this commit. Stripe just released this for client-side libraries so we can get a clear idea on the usage of the library. Can you add this back (or maybe I am missing where you put it)?

Reply:
Let me take a look and see if I blasted it away, my bad